src/docs/releases/2.1.0.md

```diff
@@ -4,6 +4,31 @@ Release date: Not yet released
 
 ## Change Logs
 
+### External User Authentication Feature
+
+A new feature, **External User Authentication**, has been added. This feature was separated from the existing **Users** feature to provide better dependency management and offer an option to disable external authentication by default.
+
+After upgrading the site, a fast-forward option allows you to enable this feature automatically when needed. As a result, the following settings have been relocated to new classes:
+
+- The properties `UseScriptToSyncRoles` and `SyncRolesScript` have been moved from `LoginSettings` to a new class, `ExternalUserRoleLoginSettings`.
+- The property `UseExternalProviderIfOnlyOneDefined` has been moved from `LoginSettings` to `ExternalUserLoginSettings`.
+
+Additionally, the following properties have been moved from `RegistrationSettings` to `ExternalAuthenticationSettings`:
+
+- `NoPasswordForExternalUsers` is now `ExternalAuthenticationSettings.NoPassword`.
+- `NoUsernameForExternalUsers` is now `ExternalAuthenticationSettings.NoUsername`.
+- `NoEmailForExternalUsers` is now `ExternalAuthenticationSettings.NoEmail`.
+- `UseScriptToGenerateUsername` is now `ExternalAuthenticationSettings.UseScriptToGenerateUsername`.
+- `GenerateUsernameScript` is now `ExternalAuthenticationSettings.GenerateUsernameScript`.
+
+The `UsersCanRegister` property in the `RegistrationSettings` class has been marked obsolete, and a new property, `AllowSiteRegistration`, has been introduced. To enable site registration, set `AllowSiteRegistration` to `true`. For external registration, ensure the **External User Authentication** feature is enabled.
+
+!!! note
+    When configuring `LoginSettings` or `RegistrationSettings` in a recipe, ensure that the recipe's properties are updated to reflect the new settings classes.
+
+!!! note
+    When the **External User Authentication** feature is enabled, the `/Register` endpoint will always become available.
```

Comment:
Move this up after the first two paragraphs, because this is more important than the property changes that should affect very few people (only when you programmatically interact with these settings).

Reply:
done